### PR TITLE
Corrected batch norm docs with the exact computations of the standard deviation

### LINKED ISSUE #77427: Batch norm doc mentions using biased variance estimator, but it actually uses the unbiased variance estimator
### 📚 The doc issue

The [BatchNorm documentation](https://pytorch.org/docs/stable/generated/torch.nn.BatchNorm2d.html) clearly states:
> The standard-deviation is calculated via the biased estimator, equivalent to `torch.var(input, unbiased=False)`.

However, when looking at the computations, it appears to use the *unbiased* estimator as shown with the following code:

```python
import numpy as np
import torch
from torch.nn import BatchNorm2d, init

np.random.seed(0)
inputs_2d = np.random.normal(size=(16, 3, 8, 8)).astype(np.float32)

bn_momentum = 0.

offline_bn_torch = BatchNorm2d(3, eps=1e-18, momentum=1-bn_momentum)
init.constant_(offline_bn_torch.weight, 1)
init.constant_(offline_bn_torch.bias, 0)
offline_bn_torch.reset_running_stats()
print(list(offline_bn_torch.buffers())[1])
offline_bn_torch.train()
offline_bn_torch(torch.tensor(inputs_2d))
print(list(offline_bn_torch.buffers())[1])
inputs_2d = np.transpose(inputs_2d, (0, 2, 3, 1))
inputs_2d = np.reshape(inputs_2d, (-1, 3))
print(np.std(inputs_2d, axis=0)**2)
print(torch.var(torch.tensor(inputs_2d), 0, unbiased=True, keepdim=False))
print(torch.var(torch.tensor(inputs_2d), 0, unbiased=False, keepdim=False))
```

which outputs:
```
tensor([1., 1., 1.])
tensor([0.9247, 0.9724, 0.9275])
[0.9238283  0.9714646  0.92655975]
tensor([0.9247, 0.9724, 0.9275])
tensor([0.9238, 0.9715, 0.9266])
```

### Suggest a potential alternative/fix

The documentation should actually say:

> The standard-deviation is calculated via the unbiased estimator, equivalent to torch.var(input, unbiased=True).

I could submit a PR if needed.

cc @svekars @holly1238 @albanD @mruberry @jbschlosser @walterddr @kshitij12345

### PR DESCRIPTION
**Pull request by zaccharieramzi**
Fixes #77427

@jbschlosser sory for taking so long to submit this, I just realized this had been sitting in my backlog for too long.

### CODE REVIEW

**Review by albanD — APPROVED** [review]
Thanks for taking the time to send the PR! Looks good!
